sensor/common/scannerclient/grpc_client_test.go

```diff
@@ -68,9 +68,8 @@ func Test_v4Client_GetImageAnalysis(t *testing.T) {
 				cfg: &types.Config{},
 			},
 			want: &ImageAnalysis{
-				ScanStatus:     scannerV1.ScanStatus_SUCCEEDED,
-				V4Contents:     &v4.Contents{},
-				IndexerVersion: v4IndexerVersion,
```

Comment:
Not sure how to add this back with the `CallOption` change. Could export everything from `pkg/scannerv4/client/client.go` and call the `.Do()` method on the mock, but I don't really know what we're testing at that point.